olga/charts/tests/test_views.py

```diff
@@ -0,0 +1,269 @@
+"""
+Tests for charts views.
+"""
+
+import json
+from datetime import datetime
+
+from mock import patch
+
+from django.test import TestCase
+
+from olga.charts.views import (
+    MapView,
+    GraphsView,
+    get_data_created_datetime_scope
+)
+
+# pylint: disable=invalid-name
+
+
+class TestMapView(TestCase):
+    """
+    Tests for map view, that contains world map statistics functionality.
+    """
+
+    def setUp(self):
+        """
+        Set up common response.
+        """
+        self.response = self.client.get('/map/')
+
+    def test_map_url(self):
+        """
+        Verify that map url resolves to map view.
+        """
+        self.assertEqual(self.response.resolver_match.func.__name__, MapView.__name__)
+
+    def test_map_view_status_code(self):
+        """
+        Verify that map view returns status 200 (ok) after get request to map url.
+        """
+        self.assertEqual(self.response .status_code, 200)
+
+    def test_map_view_html(self):
+        """
+        Verify that map view uses correct html-template after get request to map url.
+        """
+        self.assertTemplateUsed(self.response, 'charts/worldmap.html')
+
+    def test_map_view_context_fields(self):
+        """
+        Verify that map view render correct context fields.
+        """
+        context_fields = [
+            'datamap_countries_list',
+            'tabular_countries_list',
+            'top_country',
+            'countries_amount',
+            'first_datetime_of_update_data',
+            'last_datetime_of_update_data',
+        ]
+
+        for field in context_fields:
+            self.assertIn(field, self.response .context)
```

Comment:
> Again, you testing field names but not field values. When you will be testing field values, you already will be testing field names, right?

Rule I need to test all by part costs me this unnecessary code.
I agree with you.